src/components/button/button.js

```diff
@@ -57,9 +53,12 @@ Button.propTypes = {
   action: PropTypes.func,
   children: PropTypes.node,
   disabled: PropTypes.bool,
-  extraClasses: PropTypes.string,
```

Comment:
The GUI uses this extensively to add additional classes so we don't want to remove it, just add it to the `classnames` function call in the render method.